src/app/features/section/shared/list/member-list/member-list.component.ts

```diff
@@ -0,0 +1,47 @@
+import { ChangeDetectionStrategy, Component, inject, OnInit } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { ActivatedRoute } from '@angular/router';
+import { Observable } from 'rxjs';
+import { Store } from '@ngrx/store';
+import { MatListModule } from '@angular/material/list';
+
+import * as sectionSelectors from '../../../store/section.selectors';
+import { MemberListItemComponent } from '../list-item/member-list-item.component';
+import { User } from '../../interfaces';
+
+@Component({
+  selector: 'app-members-list',
+  standalone: true,
+  imports: [MatListModule, MemberListItemComponent, CommonModule],
+  templateUrl: './member-list.component.html',
+  styleUrls: ['./member-list.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MemberListComponent implements OnInit {
+  sectionId!: number;
+  usersOfSection$!: Observable<User[] | undefined>;
```

Comment:
Ta sama sytuacja - jeśli oczekujemy jakiegoś input'a i nie możemy bez niego wyświetlić komponentu to uwzględnijmy go w selektorze :)